luda/editor/client/src/sequenceListView/sequenceList/generateSequenceListRows/renderPreviewSlider.ts

```diff
@@ -0,0 +1,87 @@
+import {
+  ColorUtil,
+  FontWeight,
+  Language,
+  Rect,
+  Render,
+  Text,
+  TextAlign,
+  TextBaseline,
+} from "namui";
+import { SequenceListViewState } from "../../type";
+
+export const renderPreviewSlider: Render<
+  SequenceListViewState,
+  {
+    width: number;
+    height: number;
+  }
+> = (state, props) => {
+  const { width, height } = props;
+  const { preloadedSequence } = state;
+
+  if (!preloadedSequence) {
+    return;
+  }
+
+  const loadable = !preloadedSequence.isLoading && preloadedSequence.isSequence;
+
+  return [
+    Rect({
+      x: 0,
+      y: 0,
+      width,
+      height,
+      style: {
+        fill: {
+          color: ColorUtil.Grayscale01(0.5),
+        },
+        round: {
+          radius: height / 2,
+        },
+      },
+      onMouseMoveIn: (event) => {
+        if (preloadedSequence.isLoading || !preloadedSequence.isSequence) {
+          return;
+        }
+        preloadedSequence.seekerMs =
+          (preloadedSequence.lengthMs * event.translated.x) / width;
+      },
```

Comment:
Even if we don't click, seek preview according to the mouse position.
Would it be better to click and drag for seeking preview, like a slider?